### PR TITLE
Spellchecking on markdown and text files

### PR DESCRIPTION
**Pull request by nikcio**
Hi!

I just ran some simple spellchecking on the markdown and text files I found 😄

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -23,10 +23,10 @@ There are several [**Open Issues on GitHub**](https://github.com/apache/lucenene
...
 * Making demos and tutorials, blogging about Lucene.Net, etc. (and providing feedback on how we can make the API better!). If you write a helpful Lucene.Net post on your blog, be sure to let us know so we can link to it.
 * Helping out with documentation. We are still trying to make the API docs easily navigable (see #206), and there are many files that are not formatted correctly (links not appearing, tables not very readable, etc). Also, we need help getting all of the Java-related documentation converted to use .NET methodologies.
-* Fixing TODOs. There are several TODOs throughout the code that need to be reviewed and action taken, if necessary. Search for `LUCENENET TODO|LUCENE TO-DO` using the regular expression option in Visual Studio to find them. Do note there are a lot of TODOs left over from Java Lucene that are safe to ignore.
+* Fixing TODOs. There are several TODOs throughout the code that needs to be reviewed and action taken, if necessary. Search for `LUCENENET TODO|LUCENE TO-DO` using the regular expression option in Visual Studio to find them. Do note there are a lot of TODOs left over from Java Lucene that are safe to ignore.
```
Comment (NightOwl888) [inline CONTRIBUTING.md:29]: "need" is the correct tense here. We are referencing the TODOs (plural) rather than the code.

File: README.md [hunk 2]
```diff
@@ -238,12 +238,12 @@ The following options are case-insensitive. Each option has both a short form in
     <tr>
         <td>&#8209;fv</td>
         <td>&#8209;&#8209;file-version</td>
-        <td>The assembly file version. If not supplied, will use the --package-version (excluding any pre-release tag). The assembly version is generated as the major version of this value without the minor, build, or revision compoenents.</td>
+        <td>The assembly file version. If not supplied, will use the --package-version (excluding any pre-release tag). The assembly version is generated as the major version of this value without the minor, build, or revision components.</td>
```
Comment (NightOwl888) [inline README.md:241]: Please change this to read "The assembly version will be derived from the major version component of the passed in value, excluding the minor, build and revision components."

File: README.md [hunk 3]
```diff
@@ -238,12 +238,12 @@ The following options are case-insensitive. Each option has both a short form in
...
         <td>build&nbsp;&#8209;pv 4.8.0&#8209;beta00001&nbsp;&#8209;fv 4.8.0</td>
     </tr>
 </table>
 
-For example the following command creates a Release build with NuGet package version 4.8.0‑ci00015 and file version 4.8.0. Assembly version will be equal to the passed in major version (in this case 4.0.0).
+For example, the following command creates a Release build with NuGet package version 4.8.0‑ci00015 and file version 4.8.0. The assembly version will be equal to the value passed in the major version (in this case 4.0.0).
```
Comment (NightOwl888) [inline README.md:246]: Please change this to read "The assembly version will be derived from the major version component of the passed in value, excluding the minor, build and revision components (in this case 4.0.0)."

File: ACKNOWLEDGEMENTS.txt [hunk 4]
```diff
@@ -7,6 +7,6 @@ were developed by Martin Porter and Richard Boulton.
 The full snowball package is available from http://snowball.tartarus.org/
 
 Apache Lucene.Net is a port of Jakarta Lucene to C#.  
-The port from Java to C# of version 1.4.0, 1.4.3, 1.9, 1.9.1, 2.0 and 2.1 were done 
-primary by George Aroush. To contact George Aroush please visit http://www.aroush.net/.
+The port from Java to C# of version 1.4.0, 1.4.3, 1.9, 1.9.1, 2.0 and 2.1 was done
```
Comment (NightOwl888) [inline ACKNOWLEDGEMENTS.txt:10]: Please change from "version" to "versions" and keep this as "were" instead of "was".

File: README.md [hunk 5]
```diff
@@ -90,7 +90,7 @@ PM> Install-Package Lucene.Net -Pre
 - [Lucene.Net.Benchmark](https://www.nuget.org/packages/Lucene.Net.Benchmark/) - System for benchmarking Lucene
 - [Lucene.Net.Classification](https://www.nuget.org/packages/Lucene.Net.Classification/) - Classification module for Lucene
 - [Lucene.Net.Codecs](https://www.nuget.org/packages/Lucene.Net.Codecs/) - Lucene codecs and postings formats
-- [Lucene.Net.Expressions](https://www.nuget.org/packages/Lucene.Net.Expressions/) - Dynamically computed values to sort/facet/search on based on a pluggable grammar
+- [Lucene.Net.Expressions](https://www.nuget.org/packages/Lucene.Net.Expressions/) - Dynamically computed values to sort/facet/search based on a pluggable grammar
```
Comment (NightOwl888) [inline README.md:93]: Please do not make this change. We want to keep this aligned with https://lucene.apache.org/core/4_8_0/index.html.

File: README.md [hunk 6]
```diff
@@ -57,7 +57,7 @@ PM> Install-Package Lucene.Net
 ##### All Packages
 
 - [Lucene.Net](https://www.nuget.org/packages/Lucene.Net/3.0.3) - Core library
-- [Lucene.Net.Contrib](https://www.nuget.org/packages/Lucene.Net.Contrib/3.0.3) - Various user contributed functionality and extras
+- [Lucene.Net.Contrib](https://www.nuget.org/packages/Lucene.Net.Contrib/3.0.3) - Various user-contributed functionality and extras
```
Comment (NightOwl888) [inline README.md:60]: Either don't make this change, or make the change here and in the documentation: https://github.com/apache/lucenenet/blob/04ac9c460241480c2b9e0d38f18eaf6127583223/websites/apidocs/index.md#libraries

File: README.md [hunk 7]
```diff
@@ -101,7 +101,7 @@ PM> Install-Package Lucene.Net -Pre
 - [Lucene.Net.Queries](https://www.nuget.org/packages/Lucene.Net.Queries/) - Filters and Queries that add to core Lucene
 - [Lucene.Net.QueryParser](https://www.nuget.org/packages/Lucene.Net.QueryParser/) - Text to Query parsers and parsing framework
 - [Lucene.Net.Replicator](https://www.nuget.org/packages/Lucene.Net.Replicator/)  Files replication utility
-- [Lucene.Net.Sandbox](https://www.nuget.org/packages/Lucene.Net.Sandbox/) - Various third party contributions and new ideas
+- [Lucene.Net.Sandbox](https://www.nuget.org/packages/Lucene.Net.Sandbox/) - Various third-party contributions and new ideas
```
Comment (NightOwl888) [inline README.md:104]: Either don't make this change, or make the change here and in the documentation: https://github.com/apache/lucenenet/blob/04ac9c460241480c2b9e0d38f18eaf6127583223/websites/apidocs/index.md#libraries

**Review by NightOwl888 — CHANGES_REQUESTED** [review]

File: ACKNOWLEDGEMENTS.txt (hunk 4 above)
Comment (rclabo) [reply ACKNOWLEDGEMENTS.txt:10]: The text "primary by George Aroush. To contact George Aroush please visit http://www.aroush.net/" shouldn't be removed. Please put this acknowledgement back.

**Comment by nikcio** [comment]
The PR should now be aligned with the requested changes.

File: README.md [hunk 8]
```diff
@@ -238,12 +238,12 @@ The following options are case-insensitive. Each option has both a short form in
     <tr>
         <td>&#8209;fv</td>
         <td>&#8209;&#8209;file-version</td>
-        <td>The assembly file version. If not supplied, will use the --package-version (excluding any pre-release tag). The assembly version is generated as the major version of this value without the minor, build, or revision compoenents.</td>
+        <td>The assembly version will be derived from the major version component of the passed in value, excluding the minor, build and revision components.</td>
```
Comment (NightOwl888) [inline README.md:241]: Please change to "The assembly file version. If not supplied, defaults to the --package-version value (excluding any pre-release label). The assembly version will be derived from the major version component of the passed in value, excluding the minor, build and revision components."

**Review by NightOwl888 — CHANGES_REQUESTED** [review]

File: README.md (hunk 8 above)
Comment (nikcio) [reply README.md]: I've now changed it 😄

**Review by NightOwl888 — APPROVED** [review]

**Pull request merged**